bridges/otelzerolog/hook.go

```diff
@@ -3,7 +3,26 @@
 
 // Package otelzerolog provides a [Hook], a [zerolog.Hook] implementation that
 // can be used to bridge between the [zerolog] API and [OpenTelemetry].
+// Note: otelzerolog is a basic implementation and does not support adding custom fields.
+
+// # Record Conversion
+//
+// The [zerolog.event] records are converted to OpenTelemetry [log.Record] in
+// the following way:
+//
+//   - Message is set as the Body using a [log.StringValue].
+//   - Level is transformed and set as the Severity. The SeverityText is also
+//     set.
+// The Level is transformed to the OpenTelemetry Severity types in the following way.
+//
+//   - [zerolog.DebugLevel] is transformed to [log.SeverityDebug]
+//   - [zerolog.InfoLevel] is transformed to [log.SeverityInfo]
+//   - [zerolog.WarnLevel] is transformed to [log.SeverityWarn]
+//   - [zerolog.ErrorLevel] is transformed to [log.SeverityError]
+//   - [zerolog.PanicLevel] is transformed to [log.SeverityFatal1]
+//   - [zerolog.FatalLevel] is transformed to [log.SeverityFatal2]
 // [OpenTelemetry]: https://opentelemetry.io/docs/concepts/signals/logs/
+
```

Comment:
```suggestion
//
// # Record Conversion
//
// The [zerolog.event] records are converted to OpenTelemetry [log.Record] in
// the following way:
//
//   - Message is set as the Body using a [log.StringValue].
//   - Level is transformed and set as the Severity. The SeverityText is also
//     set.
//
// The Level is transformed to the OpenTelemetry Severity types in the following way.
//
//   - [zerolog.DebugLevel] is transformed to [log.SeverityDebug]
//   - [zerolog.InfoLevel] is transformed to [log.SeverityInfo]
//   - [zerolog.WarnLevel] is transformed to [log.SeverityWarn]
//   - [zerolog.ErrorLevel] is transformed to [log.SeverityError]
//   - [zerolog.PanicLevel] is transformed to [log.SeverityFatal1]
//   - [zerolog.FatalLevel] is transformed to [log.SeverityFatal2]
//
//
// NOTE: Fields are not transformed because of https://github.com/rs/zerolog/issues/493.
//
// [OpenTelemetry]: https://opentelemetry.io/docs/concepts/signals/logs/
```

Please do not forget to double-check if the documentation is properly rendered.
Here you can find how to do it: https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md#documentation